### PR TITLE
[WIP] Add UI tests for Policy button in the physical server page

### PR DESCRIPTION
**Pull request by igorrran**
Purpose or Intent
=================
* Add new tests to cover all options in the Policy button are working on physical server details page.

Depends On: https://github.com/ManageIQ/integration_tests/pull/6898

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ab88414799b618e0276458a61d4c0d006702964c:

`cfme/tests/physical_infrastructure/test_physical_server_policy_options.py`:
- :bangbang: Line 22:101: **E501** *line too long (118 > 100 characters)*
- :bangbang: Line 28:101: **E501** *line too long (112 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7f27179318e38d872047b6141267210d70d460b1:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 71:101: **E501** *line too long (105 > 100 characters)*

`cfme/services/service_catalogs/__init__.py`:
- :red_circle: Line 6:1: **F401** *'widgetastic_patternfly.Button' imported but unused*

`cfme/services/service_catalogs/ui.py`:
- :red_circle: Line 4:1: **F401** *'widgetastic.widget.Select' imported but unused*

`cfme/tests/cloud_infra_common/test_relationships.py`:
- :red_circle: Line 130:5: **F821** *undefined name 'logger'*

`cfme/tests/cloud_infra_common/test_vm_instance_analysis.py`:
- :red_circle: Line 14:1: **F401** *'cfme.configure.configuration' imported but unused*

`cfme/tests/physical_infrastructure/ui/test_physical_server_details_buttons.py`:
- :bangbang: Line 12:101: **E501** *line too long (107 > 100 characters)*

`cfme/tests/ssui/test_ssui_myservice.py`:
- :heavy_exclamation_mark: Line 168:1: **W391** *blank line at end of file*

`cfme/utils/smem_memory_monitor.py`:
- :red_circle: Line 0:0: **EPY3** *Usage of .iteritems*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e08fdb9f9d1144c5ab42acd589a9a781732bd8ba:

`cfme/tests/physical_infrastructure/ui/test_physical_server_details_buttons.py`:
- :bangbang: Line 12:101: **E501** *line too long (107 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0ba1c0844025ff4c8e6697210427ebbbae00b9d7:

`cfme/tests/physical_infrastructure/ui/test_physical_server_details_buttons.py`:
- :heavy_exclamation_mark: Line 61:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by igorrran** [comment]
The failures in the ci checks are not related to this pull request.

File: cfme/common/physical_server_views.py [hunk 1]
```diff
@@ -161,7 +161,17 @@ class PhysicalServerManagePoliciesView(BaseLoggedInPage):
 
     @property
     def is_displayed(self):
-        return False
+        return self.entities.get_id_by_name(self.context["object"].name) is not None
+
+
+class PhysicalServerEditTagView(BaseLoggedInPage):
```
Comment (rlbabyuk) [inline cfme/common/physical_server_views.py]: Required: Use Taggable class instead https://github.com/ManageIQ/integration_tests/blob/8b25294fc81324e1d4450b7b37d30fe85f19a448/cfme/common/__init__.py#L249, this is a duplication

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
@igorrran @miha-plesko Would you like to take over this PR and rebase, addressing the open comments? Or would you like to close this PR and pursue this work elsewhere?

Thanks!

**Comment by miha-plesko** [comment]
@matejart WDYT?

**Pull request closed without merging**